gravity-forms/gw-validate-current-password.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * Gravity Wiz // Gravity Forms // Validate Current Password
+ * http://gravitywiz.com/
+ */
+// update "123" to your form ID, update "6" to your Password field ID
```

Comment:
Update this line to read:
```
// Update "123" to your form ID and "4" to your Password field ID.
```

Reply:
Done

Reply:
I'm not seeing this change?

Reply:
Oh, it looks like you changed the IDs but not the punctutation. You can just copy and paste the exact string I provided. 👌

Reply:
Sorry about that. Guess I was focused on the IDs